sdk/python/symbolchain/NetworkTimestamp.py

```diff
@@ -50,4 +50,4 @@ def to_difference(self, reference_datetime):
 		if reference_datetime < self.epoch:
 			raise ValueError('timestamp cannot be before epoch')
 
-		return (reference_datetime - self.epoch) / datetime.timedelta(**{self.time_units: 1})
+		return (reference_datetime - self.epoch) // datetime.timedelta(**{self.time_units: 1})
```

Comment:
can the comment be removed?

Reply:
not a comment - `//` is integer division